manticore/core/cpu/abstractcpu.py

```diff
@@ -1,18 +1,24 @@
-from capstone import *
-from capstone.arm import *
-from capstone.x86 import *
+import inspect
```

Comment:
Just a reordering here to put the parseargs in alphabetic order, as well as abort early if the
user does not have binary ninja installed and selected that disassemble option